### ISSUE TITLE
Too few parameter adjustments/可调节参数太少

### ISSUE THREAD

**Issue by vanch007** [jeffy5/comfyui-faceless-node#1]
Unlike facefusion, you can select models and configure various parameters.

**Comment by jeffy5** [comment]
Thanks for the feedback. These features are still under development and will be supported in the future.

**Comment by jeffy5** [comment]
Now supports switching face swapper and face restore models, and will consider opening more advanced options later.

**Comment by hesa2020** [comment]
I agree I wanted to move away from ReActor to this because I was thinking it would have all the settings facefusion has to offer.

Il be keep a close eye on this node until then.

**Comment by zin-header** [comment]
please add mouth restoration 😁😁
